src/redux/reducers/covidReducer.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable camelcase */
```

Comment:
- [ ] Let's avoid disabling linters, instead, we should learn how to fix errors that are shown to have a better app.

Reply:
@elisha2kyakpo1, I have tried to resolve this error but according to the API that I am using, the camelcase can not be solved through linters as the data coming from API are camel-cased. I have solved other linters errors but don’t mind about this 🙌

Reply:
Hi @keza681, in that case, then you can disable that specific linter.